Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,16 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+
+function toUpperSnakeCase (str){
+
+    // Convert the string to uppercase and replace spaces with underscores
+    // str.replace(/ /gi, "_") > replace all of spaces with under score
+    //.toUpperCase() is a method that capitalize all letters
+    snakeCaseString = str.replace(/ /gi, "_").toUpperCase();
```

Comment:
If we just need to replace space, do we need the `i` flag in the regular expression?